src/ng/compile.js

```diff
@@ -779,6 +779,14 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return bindings;
   }
 
+  function assertFirstLetterIsLowerCased(name) {
+    var letter = name.charAt(0);
+    if (letter !== letter.toLowerCase()) {
+      throw ngMinErr('badname', "Directive '{0}' is not a valid name. Make sure the first letter is lowercase");
```

Comment:
"low case" or "lower case" both work for me, but please make the error describe what the error is and do not make a reference to the person nor what should he/she do